src/components/Header/Header.tsx

```diff
@@ -1,5 +1,16 @@
+import { faBars } from "@fortawesome/free-solid-svg-icons";
+import { HeaderWrapper, Logo, StyledFontAwesomeIcon } from "./Header.styles";
+import Menu from "./Menu";
+import GetStartedBtn from "../GetStarted";
+import logoSVG from "../../images/logo.svg";
+
 export default function Header() {
   return (
-    <div>Header</div>
-  )
+    <HeaderWrapper>
+      <Logo src={logoSVG} alt="logo" />
+      <Menu />
+      <GetStartedBtn mobileDisplay={false} />
+      <StyledFontAwesomeIcon icon={faBars} />
```

Comment:
tutaj warto nadać bardziej szczegółową nazwę dla komponentu, bo ciężko na 1 rzut oka stwierdzić czego to dotyczy. Sprawdziłem, że jest to burger menu, więc możesz to w ogóle wyciągnąć do osobnego komponentu